src/components/ChoiceList/tests/ChoiceList.test.tsx

```diff
@@ -103,6 +103,27 @@ describe('<ChoiceList />', () => {
         expect(choiceElements.contains(children)).toBe(true);
       });
 
+      it('renders a choice with children wrapper div when choice is selected', () => {
+        const selectedIndexes = [2];
+        const selected = selectedIndexes.map((index) => choices[index].value);
+
+        choices = [
+          choices[0],
+          choices[1],
+          {
+            ...choices[2],
+            renderChildren: renderChildrenSpy,
+          },
+        ] as any;
+
+        const choiceElements = shallowWithAppProvider(
+          <ChoiceList selected={selected} choices={choices} />,
+        );
+
+        expect(renderChildrenSpy).toHaveBeenCalled();
+        expect(choiceElements.find('div').exists()).toBeTruthy();
```

Comment:
I feel your selector is a bit too permissive. Do you think something like that would be better? 
`expect(choiceElements.find('li').at(selectedIndex).find('div').exists()).toBeTruthy();`
`selectedIndex` would be `2`.

Reply:
or even with pure CSS :)

Reply:
A bit new to testing in react. (/to react) What'd that look like?

Reply:
`TypeError: Enzyme::Selector does not support the "nth-child" pseudo-element or pseudo-class selectors`

😢